packages/fetchye/src/defaultMapOptionsToKey.js

```diff
@@ -16,6 +16,7 @@
 
 export const defaultMapOptionsToKey = (options) => {
   const {
+    signal,
```

Comment:
Can you make a unit tests that calls this function with this name and shows that it is removed.

One that would fail if this change was reverted